myosuite/envs/myo/myochallenge/soccer_v0.py

```diff
@@ -0,0 +1,510 @@
+""" =================================================
+# Copyright (c) Facebook, Inc. and its affiliates
```

Comment:
Fix the copyrights

Reply:
can we change with `Copyright (c) MyoSuite Authors`